test/pylib/scylla_cluster.py

```diff
@@ -650,13 +650,13 @@ async def release_ips(self) -> None:
     async def stop(self) -> None:
         """Stop all running servers ASAP"""
         if self.is_running:
+            self.is_running = False
```

Comment:
Should we so the same for `stop_gracefully` to make the code consistent?

Reply:
Done in https://github.com/scylladb/scylladb/commit/3388d61858baf0dedd07666ade3047cbc4c413d1